server/src/main/java/org/opensearch/plugins/PluginInfo.java

```diff
@@ -65,17 +73,22 @@ public class PluginInfo implements Writeable, ToXContentObject {
 
     public static final String OPENSEARCH_PLUGIN_PROPERTIES = "plugin-descriptor.properties";
     public static final String OPENSEARCH_PLUGIN_POLICY = "plugin-security.policy";
+    private static final JsonFactory jsonFactory = new JsonFactory();
 
     private final String name;
     private final String description;
     private final String version;
-    private final Version opensearchVersion;
+    private final List<SemverRange> opensearchVersionRanges;
     private final String javaVersion;
     private final String classname;
     private final String customFolderName;
     private final List<String> extendedPlugins;
     private final boolean hasNativeController;
 
+    static {
```

Comment:
I don't think you need this block, `JsonFactory` could be initialized in one shot

Reply:
Updated